src/libraries/System.DirectoryServices.AccountManagement/tests/PrincipalTest.cs

```diff
@@ -1,143 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System.Collections.Generic;
-using System.Security.Principal;
-using Xunit;
-
-namespace System.DirectoryServices.AccountManagement.Tests
-{
-    public abstract class PrincipalTest : IDisposable
-    {
-        public PrincipalContext DomainContext { get; private set; }
-
-        public PrincipalTest() => RefreshContext();
-
-        private void RefreshContext()
-        {
-            string username = "Administrator";
-            string password = Environment.GetEnvironmentVariable("TESTPASSWORD");
-            string OU = "Tests";
-            string baseDomain = WindowsIdentity.GetCurrent().Name.Split(new char[] { '\\' })[1] + "-TEST";
-            string domain = $"{baseDomain}.nttest.microsoft.com";
-            string container = $"ou={OU},dc={baseDomain},dc=nttest,dc=microsoft,dc=com";
-            DomainContext?.Dispose();
-            try
-            {
-                DomainContext = new PrincipalContext(ContextType.Domain, domain, container, username, password);
-            }
-            catch
-            {
-            }
-        }
-
-        public void Dispose() => DomainContext?.Dispose();
-
-        [Fact]
-        public void AddExistingPrincipal()
```

Comment:
Looks like we are missing all of the base tests as well. I don't see these getting migrated over, only the ctor tests seemed to be moved.

Reply:
Most of them were already part of the new test suite:
- `AddExistingPrincipal`
-- Tests adding a principal - already in new suite: https://github.com/dotnet/runtime/blob/0ea9425bcf527b615a07f6cf5fe9936b6070cc0a/src/libraries/System.DirectoryServices.AccountManagement/tests/AccountManagementTests.cs#L102
-- Tests adding duplicate principal - already in new suite: https://github.com/dotnet/runtime/blob/0ea9425bcf527b615a07f6cf5fe9936b6070cc0a/src/libraries/System.DirectoryServices.AccountManagement/tests/AccountManagementTests.cs#L292
- `TestExtendedPrincipal` I pulled apart from this PR into another PR for a bugfix (#56696), as writing those tests properly causes a bug to trigger that I fixed there. I did not want to bring in breaking tests here and have that PR depend on this one. I could move things around a bit though, if desired.
- `Save_ThrowsInvalidOperationException` tries to store a principal using an "incorrect" context (different from the creating context as well); which I migrated to https://github.com/dotnet/runtime/blob/0ea9425bcf527b615a07f6cf5fe9936b6070cc0a/src/libraries/System.DirectoryServices.AccountManagement/tests/AccountManagementTests.cs#L319